api/MessagePort.json

```diff
@@ -0,0 +1,324 @@
+{
+  "api": {
+    "MessagePort": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MessagePort",
+        "support": {
+          "webview_android": {
+            "version_added": "4"
+          },
+          "chrome": {
+            "version_added": "4"
+          },
+          "chrome_android": {
+            "version_added": "4"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": true,
+              "version_removed": "41"
+            },
+            {
+              "version_added": "41",
+              "notes": "Available in workers."
```

Comment:
I think this could be a pretty good way of signalling availability in workers, but do we need to do this also with (say) Chrome somehow? @Elchi3 thoughts?